travis-ci/vmtest/helpers.sh

```diff
@@ -1,26 +1,18 @@
 # $1 - start or end
 # $2 - fold identifier, no spaces
 # $3 - fold section description
-travis_fold() {
+foldable() {
   local YELLOW='\033[1;33m'
   local NOCOLOR='\033[0m'
-  if [ -z ${GITHUB_WORKFLOW+x} ]; then
-    echo travis_fold:$1:$2
+  if [ $1 = "start" ]; then
+    line="::group::$2"
     if [ ! -z "${3:-}" ]; then
-      echo -e "${YELLOW}$3${NOCOLOR}"
+      line="$line - ${YELLOW}$3${NOCOLOR}"
     fi
-    echo
   else
```

Comment:
can be done as a separate PR

I wonder if it will make sense to either explicitly check for "end" in here or have two separate functions to avoid if and need to pass one of the parameters

Reply:
We will also need to backport this to libbpf/ci code or merge them somehow

Reply:
Thanks for taking a look.

> can be done as a separate PR

It's a separate commit. What would be the benefit of having a separate pull request?

> I wonder if it will make sense to either explicitly check for "end" in here or have two separate functions to avoid if and need to pass first parameter

It has nothing to do with this pull request's intention, though? I am sure there was a reason it was written this way and I don't see a good reason to change it, to be honest.

Reply:
> We will also need to backport this to libbpf/ci code or merge them somehow

It has already been merged there a while back: https://github.com/libbpf/ci/pull/18

Reply:
> 

my concern is that if you pass any garbage as a second argument to the function right now it will always behave as closing the fold, third argument does not do anything in this case as well. it is not related to this change, hence I proposed to consider it as a separate PR unrelated to this